pages/about.html

```diff
@@ -5,8 +5,71 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="./styles/about.css" rel="stylesheet">
 </head>
+
 <body>
+  <header>
+    <h1>About me!</h1>
+    <p id="my-intro">
+      My name is <b>Pavi Rannulu.</b> 
+      <br>
+      Pronouns:   They/Them/Theirs
+      <br>
+      I am queer, non-binary, brown, and an immigrant.
+      <br>
+      Best known for: having big heart & a curious mind. 
+    </p>
+
+
+  </header>
+ 
+  <section class = "chonk">
+  <h2>My journey from Sri Lanka to San Diego</h2>
+  <p>
+  I was born in Colombo, Sri Lanka. It is a beautiful island with a troubled past, riddled with civil unrest, conflict and economic troubles.
+   At the age of 17, I moved to the United States of America by myself to pursue my undergraduate studies at the University of Maryland.
+  There, I had the priviledge of studying two subjects that have always peaked my interest: mathematics and physics. 
+  After graduating with my Bachelors with Honors in 2018, I moved to San Diego, CA, to pursue my first full-time role. I was a Student Services Specialist at Art of Problem Solving, an online math school.
+  I worked there for over 3 years, while simultaneously cultivating a community of queer and trans folks in the local area.
+  In February 2022, I left my  job to explore how to be more intentionally involved in this community and to figure out my next career steps.
+  My passion for problem solving and social justice led me to Ada Developers Academy, where I started learning coding with like-minded folks around the country.
+  </p>
+  </section>
+
+  <section class = "chonk">
```

Comment:
Based on HTML styling guide, tag attributes should not have spaced around the `=`
```html
  <section class="chonk">
```